database/seeders/VisitPageSeeder.php

```diff
@@ -12,7 +12,7 @@ public function run(): void
 
         $visitPage->translate('en')->visit_hour_intro = 'The Art Institute reopens on July 30, and we\'re so happy to welcome you back to our galleries. Please see below for new hours—including member-only hours—and updated safety policies.';
```

Comment:
@zachgarwood FYI, in WEB-2174 we removed translations from the Visit page. So references to $visitPage attributes should be direct, e.g., `$visitPage->visit_hour_subheader`. There should no longer be references to attributes via the translation helper like `$visitPage->translate('en')->visit_hour_subheader`. This code is probably representative of how things were structured before that change.

Reply:
I've removed those translate functions.